apps/client/src/pages/LocationPage.js

```diff
@@ -67,14 +83,16 @@ const LocationPage = ({ topic }) => {
         <LocationFinder
           setAddress={setAddress}
           setErrorMessage={setErrorMessage}
-          postalCode={context.address?.postalCode}
-          houseNumberFull={context.address?.houseNumberFull}
-          houseNumber={context.address?.houseNumberFull}
+          postalCode={sessionAddress?.postalCode}
```

Comment:
Can we make sure sessionAddress is an `{}`?

Reply:
Solved